scripts/tapir/deploy_bqeth.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/python3
+
+from ape import project
+
+from deployment.constants import (
+    CONSTRUCTOR_PARAMS_DIR, ARTIFACTS_DIR,
+)
+from deployment.params import Deployer
+from deployment.registry import merge_registries
+
+VERIFY = False
+CONSTRUCTOR_PARAMS_FILEPATH = CONSTRUCTOR_PARAMS_DIR / "tapir" / "bqeth.yml"
+TAPIR_REGISTRY = ARTIFACTS_DIR / "tapir.json"
+
+
+def main():
+    deployer = Deployer.from_yaml(filepath=CONSTRUCTOR_PARAMS_FILEPATH, verify=VERIFY)
+
+    global_allow_list = deployer.deploy(project.GlobalAllowList)
```

Comment:
Is this intentional? I think there should be only one `GlobalAllowList` contract that gets reused...unless I'm mistaken (cc @vzotova ). If so, deployment of the `GlobalAllowList` contract should be a separate script.

Reply:
[895cb1a](https://github.com/nucypher/nucypher-contracts/pull/308/commits/895cb1a10cb131901619de055b4f7f2555a518e1)